### PR TITLE
SPCTR-628 Fix: block descriptions updated

### PR DESCRIPTION
**Pull request by jamyrbsf**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-628

### Types of changes
Fix: block descriptions updated

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/faq-child/block.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { __ } from '@wordpress/i18n';
 
 registerBlockType( 'uagb/faq-child', {
 	title: __( 'FAQ Child', 'ultimate-addons-for-gutenberg' ),
-	description: __( 'This block helps you add single FAQ.', 'ultimate-addons-for-gutenberg' ),
+	description: __( 'Add a frequently asked quesion/accordian to display information.', 'ultimate-addons-for-gutenberg' ),
```
Comment (Avneeshv18) [inline src/blocks/faq-child/block.js:16]: There's a typo:

question/accordion

**Review by Avneeshv18 — CHANGES_REQUESTED** [review]

File: src/blocks/faq/block.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const withFaq = createHigherOrderComponent( ( BlockEdit ) => {
 
 registerBlockType( 'uagb/faq', {
 	title: __( 'FAQ', 'ultimate-addons-for-gutenberg' ),
-	description: __( 'This block helps you add FAQ or Accordion with automatically adding FAQ Schema to your page.', 'ultimate-addons-for-gutenberg' ),
+	description: __( 'Add accordians/FAQ schema to your page.', 'ultimate-addons-for-gutenberg' ),
```
[the remark this answers is not in the document]
Comment (jamyrbsf) [reply src/blocks/faq/block.js:45]: fixed it.

File: src/blocks/testimonial/block.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ import { registerBlockType } from '@wordpress/blocks';
 
 registerBlockType( 'uagb/testimonial', {
 	title: __( 'Testimonials', 'ultimate-addons-for-gutenberg' ), // Block title.
-	description: __( 'This block helps your display some amazing client feedback within your website.', 'ultimate-addons-for-gutenberg' ), // Block description.
+	description: __( 'Display customer testimonials with customizable layouts.', 'ultimate-addons-for-gutenberg' ), // Block description.
```
[the remark this answers is not in the document]
Comment (jamyrbsf) [reply src/blocks/testimonial/block.js:17]: fixed it.

File: src/blocks/faq-child/block.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamyrbsf) [reply src/blocks/faq-child/block.js:16]: fixed it.

**Review by Avneeshv18 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/forms/child-blocks/checkbox/block.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ import { __ } from '@wordpress/i18n';
 
 registerBlockType( 'uagb/forms-checkbox', {
 	title: __( 'Checkbox', 'ultimate-addons-for-gutenberg' ),
-	description: __( 'This block helps to add Checkbox field.', 'ultimate-addons-for-gutenberg' ),
+	description: __( 'Add checkboxes to allow mulitple choices from optons.', 'ultimate-addons-for-gutenberg' ),
```
Comment (Avneeshv18) [inline src/blocks/forms/child-blocks/checkbox/block.js:15]: "Options" spelling needs to be replaced.

Add checkboxes to allow multiple choices from options.

File: src/blocks/timeline/content-timeline-child/block.js [hunk 5]
```diff
@@ -16,7 +16,7 @@ import { registerBlockType } from '@wordpress/blocks';
 
 registerBlockType( 'uagb/content-timeline-child', {
 	title: __( 'Content Timeline Child', 'ultimate-addons-for-gutenberg' ),
-	description: __( 'The Timeline block lets you create beautiful timelines on your website.', 'ultimate-addons-for-gutenberg' ),
+	description: __( 'Add and customize displaying content of this timeline.', 'ultimate-addons-for-gutenberg' ),
```
Comment (Avneeshv18) [inline src/blocks/timeline/content-timeline-child/block.js:19]: Please replace it with:-

Add and customize content of this timeline.

File: src/blocks/forms/child-blocks/checkbox/block.js (hunk 4 above)
Comment (jamyrbsf) [reply src/blocks/forms/child-blocks/checkbox/block.js]: resolved this.

File: src/blocks/timeline/content-timeline-child/block.js (hunk 5 above)
Comment (jamyrbsf) [reply src/blocks/timeline/content-timeline-child/block.js]: resolved this.

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**